crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -260,7 +285,7 @@ impl Format<PyFormatContext<'_>> for FormatDanglingOpenParenthesisComments<'_> {
             write!(
                 f,
                 [
-                    line_suffix(&format_args!(space(), space(), format_comment(comment))),
+                    line_suffix(&format_args!(space(), space(), format_comment(comment)), 0),
```

Comment:
What's the reason for excluding the line width here or is this just for the moment?

Reply:
I started by isolating the change to the `ruff_formatter` crate. So, yea, this is likely leftover from that. Thanks for pointing it out.

Reply:
Added a `measure_text` helper in 686f3da95e45326e45ac2237b0e6322c3c89f78a.